### ISSUE TITLE
[Feature request] Multi-path option in payment screen

### ISSUE THREAD

**Issue by Darth-Coin** [hsjoberg/blixt-wallet#426]
I tried to do a test payment with the option "multi-path" activated in Blixt options.
Had 3 channels with enough balance to send, but payment only through one.
2 channels with Blixt node, one ch with my own node.

Maybe would be useful a "use MPP" check box in the payment screen, that can split the total amount in 2-3 parts for each available channel, or even choose the number of parts the user want to split.

**Comment by eliotcougar** [comment]
I agree... I really like having this option in Zeus wallet... When you make the payment you basically have most of the options you have in payinvoice command: fee limit, first hop, last hop, attempt MPP, attempt AMP, max parts, and max shard size...

Probing possible fee, however, is not straightforward... You could keep track of the maximum total fee seen while searching for a path, but that doesn't mean that you won't find a path with a smaller fee the second time you try to pay the invoice... I often had better luck finding a low fee path for a circular balancing payments (they are often 3+ hops) when I increased the timeout from the default 1 minute to 2-3 minutes instead of increasing the max fee...

I know that the aim is to have a UI that is uncluttered and easy to understand for non-technical people, so this kind of functionality may be hidden behind some sort of "Enable advanced features" flag in settings...
